File: repos_estudiantes_TARQUI_tercera_entrega/SIÑANI_QUELALI_IVAN/tarea3/ge1.java
public class Caja<T> {
    private T objeto; 

    public void guardar(T objeto) {
        this.objeto = objeto;
    }

    public T obtener() {
        return objeto;
    }
}

public class Main {
    public static void main(String[] args) {

        Caja<Integer> cajaEntera = new Caja<>();
        cajaEntera.guardar(42);
        System.out.println("Contenido de cajaEntera: " + cajaEntera.obtener());

        Caja<String> cajaTexto = new Caja<>();
        cajaTexto.guardar("Hola, mundo!");
        System.out.println("Contenido de cajaTexto: " + cajaTexto.obtener());
    }
}
//DIAGRAMA
//Clase Caja<T>
//---------------------------
//- objeto: T
//---------------------------
//+ guardar(T objeto): void
//+ obtener(): T
//---------------------------

//Clase Main
//---------------------------
//+ main(String[] args): void
//---------------------------

File: repos_estudiantes_TARQUI_tercera_entrega/SIÑANI_QUELALI_IVAN/tarea3/ge1.py
class Caja:
    def __init__(self):
        self.objeto = None 

    def guardar(self, objeto):
        self.objeto = objeto

    def obtener(self):
        return self.objeto

cajaenteros = Caja()
cajaenteros.guardar(42)
print("Contenido de caja-enteros:", cajaenteros.obtener())

cajatexto = Caja()
cajatexto.guardar("Hola mundo")
print("Contenido de caja-texto:", cajatexto.obtener())
